### PR TITLE
DCO

### PR DESCRIPTION
**Pull request by EthanPERRUZZA**

### CODE REVIEW

File: content/docs/guides/contribute/contribute-code/commit-conventions.en.md [hunk 1]
```diff
@@ -46,4 +48,32 @@ Welcome to ease review, but do not merge:
...
+
+To sign off a commit, simply add the `-s` flags to your `git commit` command,
+like so:
+
+```bash
+git commit -s -m "Your commit message"
+```
+This also applies when using the `git revert` command.
+
+### How to do sign a commint using git in Visual Studio Code (VS Code) ?
```
Comment (ElysaSrc) [inline content/docs/guides/contribute/contribute-code/commit-conventions.en.md:71]: Commint, petite typo

File: content/docs/guides/contribute/contribute-code/commit-conventions.en.md [hunk 2]
```diff
@@ -46,4 +48,32 @@ Welcome to ease review, but do not merge:
...
+
+Now, go in `Files` -> `Preferences` -> `Settings` and search for **Always Sign
+Off**. (Thisonly has to be done once per repository.)
```
Comment (ElysaSrc) [inline content/docs/guides/contribute/contribute-code/commit-conventions.en.md:74]: Petit espace entre this et only?

**Review by ElysaSrc — APPROVED** [review]
Careful few typos here and there but that seems good to me.

Ping @flomonster for review if this matches what was expected on the website.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+This is the website repo. For more info on the project or to contribute to the
+project itself checkout the [Contributing](https://github.com/OpenRailAssociation/osrd/blob/dev/CONTRIBUTING.md)
+file of the main repo.
+
+## Contribute code
+
+To comply with the [DCO](http://developercertificate.org/), all commits must
+include a Signed-off-by line. You can find more information about this here:
+[osrd.fr/docs/guides/contribute/contribute-code/commit-conventions/#the-developer-certificate-of-origin](https://osrd.fr/en/docs/guides/contribute/contribute-code/commit-conventions/#the-developer-certificate-of-origin)
```
Comment (flomonster) [inline CONTRIBUTING.md:29]: We can keep the full link

```suggestion
https://osrd.fr/en/docs/guides/contribute/contribute-code/commit-conventions/#the-developer-certificate-of-origin
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+### Pull request / code review
+
+[osrd.fr/docs/contribute/code/#share-your-changes](https://osrd.fr/en/docs/guides/contribute/code/#share-your-changes).
```
Comment (flomonster) [inline CONTRIBUTING.md:33]:
```suggestion
https://osrd.fr/en/docs/guides/contribute/code/#share-your-changes
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+### Report and solve bugs
+
+[osrd.fr/docs/contribute/bug-reports](https://osrd.fr/en/docs/guides/contribute/bug-reports)
```
Comment (flomonster) [inline CONTRIBUTING.md:37]:
```suggestion
https://osrd.fr/en/docs/guides/contribute/bug-reports
```

File: content/docs/guides/contribute/contribute-code/commit-conventions.en.md [hunk 6]
```diff
@@ -46,4 +48,32 @@ Welcome to ease review, but do not merge:
...
+
+```bash
+git commit -s -m "Your commit message"
+```
+This also applies when using the `git revert` command.
+
+### How to do sign a commit using git in Visual Studio Code (VS Code) ?
+
+Now, go in `Files` -> `Preferences` -> `Settings` and search for **Always Sign
+Off**. (This only has to be done once per repository.)
```
Comment (flomonster) [inline content/docs/guides/contribute/contribute-code/commit-conventions.en.md:74]: This sentence seems strange to me. What did you mean by that?

Note that you have two kind of settings in vs code. One is for the **user** (global to vs code). The other one is local to the **workspace** (which is not the same as the repository)

**Review by flomonster — COMMENTED** [review]
Thanks for your pull requests. It looks good to me overal, just a few comments.

File: content/docs/guides/contribute/contribute-code/commit-conventions.en.md (hunk 6 above)
Comment (EthanPERRUZZA) [reply content/docs/guides/contribute/contribute-code/commit-conventions.en.md:74]: I meant the workspace but after checking again it appears that this setting is bind to the user. Thanks for catching that mistake.

Comment (flomonster) [reply content/docs/guides/contribute/contribute-code/commit-conventions.en.md]: I checked and this setting (like all of them) can be bind to the workspace.

image

**Review by flomonster — APPROVED** [review]
LGTM

**Pull request merged**